roles/openshift_sanitize_inventory/tasks/deprecations.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- name: Check for usage of deprecated variables
+  debug: msg="{{ __deprecation_header }} {{ item }} is a deprecated variable and will be no longer be used in the next minor release. Please update your inventory accordingly."
+  when:
+  - lookup('env', item) | bool
+  with_items: "{{ __warn_deprecated_vars }}"
```

Comment:
Oh yeah, that's not multiple includes, that's just the same thing running for each host. That's hard to deduplicate, and arguably you don't want to since different hosts may have different variables set.

I don't think *pause* will run once for each host though... does it?